File: Datos abstractos 28-08-25/datos_abstractos.cs
using System;

class Persona {
    public string Nombre;
    public string Apellido1;
    public string Apellido2;
}

class Program {
    static void Main() {
        Persona p = new Persona();
        p.Nombre = "Pablo";
        p.Apellido1 = "Ruelas";
        p.Apellido2 = "López";

        Console.WriteLine($"{p.Nombre} {p.Apellido1} {p.Apellido2}");
    }
}

File: Datos abstractos 28-08-25/datos_abstractos.cpp
#include <iostream>
#include <string>
using namespace std;

struct Persona {
    string nombre;
    string apellido1;
    string apellido2;
};

int main() {
    Persona p;
    p.nombre = "Pablo";
    p.apellido1 = "Ruelas";
    p.apellido2 = "López";

    cout << p.nombre << " " << p.apellido1 << " " << p.apellido2 << endl;
    return 0;
}
